src/sync.js

```diff
@@ -184,26 +184,34 @@ module.exports = class Sync {
     return data.map(Sync.transformRepository);
   }
 
-  static transformCommit(data) {
+  static transformCommit(data, project) {
     const transformed = Object.assign({}, data);
     delete transformed.links;
+    transformed.project = this.transformProject(project);
     if (transformed.author.user) {
       delete transformed.author.user.links;
+      transformed.author.user.email = this.parseEmail(transformed.author.raw);
     }
     delete transformed.repository.links;
     delete transformed.summary;
     transformed.parents = data.parents.map((parent) => parent.hash);
     return transformed;
   }
 
+  static parseEmail(rawName) {
+    const startIndex = rawName.indexOf("<") + 1;
```

Comment:
I think it will break if there's no an email in the `raw` field. Please add a test to consider such scenario.